### DISCUSSION TITLE
Regular expression

### DISCUSSION THREAD

**Discussion by sangacanga1** [SubtitleEdit/subtitleedit#8440] [Q&A]
Hello, what is the regular expression that can provide the property in the example I show?
Example:
Before:
So, what's with the 'Bang-bang?'
After:
So, what's with the Bang-bang?

**Comment by niksedk** [comment]
Perhaps something like this:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/7a43113f-7b57-4857-bd66-68ba470c1bfc)

**Comment by gru123** [comment]
sangacanga1, read the tutorial (e.g. [tutorial](https://www.regular-expressions.info/tutorial.html) ), create the regex yourself and test it online [regex101](https://regex101.com/)

**Comment by ivandrofly** [comment]
Find: `'(.+\?)'` 
Replace with: $1
